docs/client_reference.rst

```diff
@@ -294,6 +294,12 @@ The client session supports the context manager protocol for self closing.
 
          .. versionadded:: 2.3
 
+      :param trace_context: Object used to give as a param for the all signals
+        triggered by the ongoing request. Default uses the object returned 
+        by the :class:`TraceConfig.trace_context()` method.
```

Comment:
:meth:`TraceConfig.trace_context` maybe. Parenthesis is not required by sphinx

Reply:
no longer required, this documentation has been deprecated.